### PR TITLE
Update meeting-policies-audio-and-video.md

### PR DESCRIPTION
**Pull request by kristofftan**
Under TAC, the "disable" value should be "Not enabled", then make this doc align with TAC's value.

### CODE REVIEW

File: Teams/meeting-policies-audio-and-video.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ This is a per-user policy. This setting controls whether audio can be turned on
 |Setting value |Behavior  |
 |---------|---------|
 |**Outgoing and incoming audio enabled**    |Outgoing and incoming audio is allowed in the meeting. This is the default setting. |
-|**Disabled**     |Outgoing and incoming audio is turned off in the meeting.     |
+|**Not enabled**     |Outgoing and incoming audio is turned off in the meeting.     |
 
 If set to **Disabled** for a user, that user can still schedule and organize meetings but they can't use audio. To join a meeting, they have to dial in through the Public Switched Telephone Network (PSTN) or have the meeting call and join them by phone. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming audio enabled** by default. On Teams mobile clients, if this setting is disabled, the user has to dial in to the meeting through the PSTN.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:49]:
```suggestion
If set to **Nor enabled** for a user, that user can still schedule and organize meetings but they can't use audio. To join a meeting, they have to dial in through the Public Switched Telephone Network (PSTN) or have the meeting call and join them by phone. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming audio enabled** by default. On Teams mobile clients, if this setting is not enabled, the user has to dial in to the meeting through the PSTN.
```

File: Teams/meeting-policies-audio-and-video.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ This is a per-user policy. This setting controls whether video can be turned on
 |Setting value |Behavior  |
 |---------|---------|
 |**Outgoing and incoming video enabled**    | Outgoing and incoming video is allowed in the meeting. This is the default setting. |
-|**Disabled**     | Outgoing and incoming video is turned off in the meeting. On Teams mobile clients, users can't share videos or photos in the meeting. <br><br>Note that if **Mode for IP audio** is disabled, then **Mode for IP video** will also remain disabled.  |
+|**Not enabled**     | Outgoing and incoming video is turned off in the meeting. On Teams mobile clients, users can't share videos or photos in the meeting. <br><br>Note that if **Mode for IP audio** is disabled, then **Mode for IP video** will also remain disabled.  |
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:64]:
```suggestion
|**Not enabled**     | Outgoing and incoming video is turned off in the meeting. On Teams mobile clients, users can't share videos or photos in the meeting. <br><br>Note that if **Mode for IP audio** is not enabled, then **Mode for IP video** will also remain not enabled.  |
```

File: Teams/meeting-policies-audio-and-video.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ This is a per-user policy. This setting controls whether video can be turned on
...
 
 If set to **Disabled** for a  user, that user can't turn on video or view videos shared by other meeting participants. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming video enabled** by default.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:66]:
```suggestion
If set to **Not enabled** for a  user, that user can't turn on video or view videos shared by other meeting participants. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming video enabled** by default.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kristofftan, kindly commit the additional suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

File: Teams/meeting-policies-audio-and-video.md [hunk 4]
```diff
@@ -44,9 +44,9 @@ This is a per-user policy. This setting controls whether audio can be turned on
...
+|**Not enabled**     |Outgoing and incoming audio is turned off in the meeting.     |
 
-If set to **Disabled** for a user, that user can still schedule and organize meetings but they can't use audio. To join a meeting, they have to dial in through the Public Switched Telephone Network (PSTN) or have the meeting call and join them by phone. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming audio enabled** by default. On Teams mobile clients, if this setting is disabled, the user has to dial in to the meeting through the PSTN.
+If set to **Nor enabled** for a user, that user can still schedule and organize meetings but they can't use audio. To join a meeting, they have to dial in through the Public Switched Telephone Network (PSTN) or have the meeting call and join them by phone. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming audio enabled** by default. On Teams mobile clients, if this setting is not enabled, the user has to dial in to the meeting through the PSTN.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:49]:
```suggestion
If set to **Not enabled** for a user, that user can still schedule and organize meetings but they can't use audio. To join a meeting, they have to dial in through the Public Switched Telephone Network (PSTN) or have the meeting call and join them by phone. Meeting participants who don't have any policies assigned (for example, anonymous participants) have this set to **Outgoing and incoming audio enabled** by default. On Teams mobile clients, if this setting is not enabled, the user has to dial in to the meeting through the PSTN.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@kristofftan, apologies, one more, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**